client/README.md

```diff
@@ -0,0 +1,68 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
I think this file is useless !!!

Reply:
I think it is useful since you can follow instructions to build and eject the app.
this is react documentation.